tla-types/src/main/scala/at/forsyte/apalache/tla/typecheck/etc/ToEtcExpr.scala

```diff
@@ -23,13 +23,16 @@ class ToEtcExpr(annotationStore: AnnotationStore, aliasSubstitution: ConstSubsti
 
   def apply(decl: TlaDecl, inScopeEx: EtcExpr): EtcExpr = {
     decl match {
-      case d: TlaConstDecl =>
-        // CONSTANT N
-        findTypeFromTagOrAnnotation(d).map(tt => mkTypeDecl(ExactRef(d.ID), d.name, tt, inScopeEx)).getOrElse(inScopeEx)
-
-      case d: TlaVarDecl =>
-        // VARIABLE x
-        findTypeFromTagOrAnnotation(d).map(tt => mkTypeDecl(ExactRef(d.ID), d.name, tt, inScopeEx)).getOrElse(inScopeEx)
+      case TlaConstDecl(_) | TlaVarDecl(_) =>
+        // CONSTANT N or VARIABLE x
+        findTypeFromTagOrAnnotation(decl) match {
+          case Some(tt) =>
+            mkTypeDecl(ExactRef(decl.ID), decl.name, tt, inScopeEx)
+
+          case None =>
+            val kind = if (decl.isInstanceOf[TlaConstDecl]) "CONSTANT" else "VARIABLE"
```

Comment:
Is there a `toString` function or `name` attribute we can use here instead?

Reply:
The name is also printed (below in the code). The standard `toString` of the case classes is not very TLAish.